molecule/commands/converge.py

```diff
@@ -25,6 +25,7 @@
 
 from molecule import ansible_playbook
 from molecule import utilities
+from molecule.ansible_galaxy_install import AnsibleGalaxyInstall
```

Comment:
Can we follow our current convention?

`from molecule import ansible_galaxy_install` in alphabetical order.